tests/fixture/Point.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Tests\Fixture;
+
+final class Point
+{
+    public function __construct(
+        public readonly float $x,
```

Comment:
asserting floats is error-prone on different systems with differend precision. Maybe better to just make it an int?

Reply:
Or a `numeric-string`, but I'm not sure how this could be a problem

Reply:
see https://www.php.net/manual/en/language.types.float.php

> So never trust floating number results to the last digit, and do not compare floating point numbers directly for equality. If higher precision is necessary, the [arbitrary precision math functions](https://www.php.net/manual/en/ref.bc.php) and [gmp](https://www.php.net/manual/en/ref.gmp.php) functions are available.